lib/location/Kconfig

```diff
@@ -0,0 +1,147 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig LOCATION
+	bool "Location"
+	depends on NRF_MODEM_LIB
+	help
+	  TODO
+
+if LOCATION
+
+config LOCATION_METHOD_GNSS
+	bool "Allow GNSS to be used for obtaining the location"
+	default y
+
+config LOCATION_METHOD_CELLULAR
+	bool "Allow cellular positioning to be used for obtaining the location"
+	select MULTICELL_LOCATION
+	default y
+
+config LOCATION_METHOD_WIFI
+	bool "Allow WiFi positioning to be used for obtaining the location"
+	depends on WIFI
```

Comment:
I'll leave a comment here to mark that we should think whether this is `select` or `depends on` or maybe `imply`. And whether `default y if WIFI` is needed.
Maybe the current approach is even the best.

Reply:
In my opinion `depends on` is fine. I think it's better that we don't start configuring WiFi stuff. Not sure if changing the default for `LOCATION_METHOD_WIFI` when `WIFI` is enabled is necessary either. If somebody with a WiFi chip doesn't want to use WiFi location, the implementation might be left enabled unintentionally.

Reply:
We'll leave this and I've removed `default y if WIFI` few lines below.